us/en/skymiles/blocks/toc/toc.js

```diff
@@ -0,0 +1,29 @@
+export default function decorate(block) {
+  const currentToc = block.querySelector('ul');
```

Comment:
Since we only cover exact matches on the headings in the 1st iteration, we could also just extract those from the DOM directly instead of having the customer manually specify those.

Alternatively, I'd leverage the internal word bookmarks for the anchors so you have label+link already done for you and then it's just a matter of applying CSS to style the list

Reply:
Just tried the second option and it works. I think this would be a natural authoring experience.

Additional Question: I was wondering what if walk through the DOM and add links to only the TOC titles that don't have any bookmarks. Or should we just leave it for the authors to manage this?

Reply:
I wouldn't overengineer for now, we can do full-auto via DOM extraction, and full-manual via anchors and bookmarks. if the authors need more, they'll tell us and we can iterate on this again as needed